GVFS/GVFS.Hooks/Program.cs

```diff
@@ -83,6 +83,12 @@ public static void Main(string[] args)
 
         private static void RunPreCommands(string[] args)
         {
+            if (ProductUpgrader.IsLocalUpgradeAvailable())

```

Comment:
What is the cost of this check? How much time are we adding to every git command now?

Reply:
Tested on GVFS repo using “Measure-Command { foreach ($i in 0..200) {git status}}”

Trials | No reminder hook | Reminder hook enabled & upgrade NOT available | Reminder Hook enabled & upgrade available
------ | ------------ | ------------- | -----------
1 | 52647.2346 | 54122.9319 | 67319.0552
2 | 66861.3138 | 64679.6913 | 63246.1476
3 | 66897.6017 | 68131.4394 | 68101.7228
4 | 62314.6312 | 67122.4883 | 68498.0093
5 | 68277.4996 | 67653.2103 | 68913.034
Average | 63399.65618 | 64341.95224 | 67215.59378
**Per call** | **317ms** | **321ms** | **336ms**